guides/common/modules/proc_using-activation-keys-for-host-registration.adoc

```diff
@@ -6,21 +6,11 @@ You can use activation keys to complete the following tasks:
 * Registering new hosts during provisioning through {ProjectName}.
 The kickstart provisioning templates in {ProjectName} contain commands to register the host using an activation key that is defined when creating a host.
 * Registering existing {RHEL} hosts.
-Configure Subscription Manager to use {ProjectServer} for registration and specify the activation key when running the `subscription-manager register` command.
 
 include::snip_host-registration-steps.adoc[]
 
 .Multiple activation keys
 You can use multiple activation keys when registering a content host.
-You can then create activation keys for specific subscription sets and combine them according to content host requirements.
-For example, the following command registers a content host to your organization with both VDC and OpenShift subscriptions:
-
-[options="nowrap" subs="+quotes"]
-----
-# subscription-manager register \
---activationkey="ak-VDC,ak-OpenShift" \
---org="_My_Organization_"
-----
```

Comment:
You can still use multiple activation keys. We could replace this with another example, such as

```
For example, you could use one activation key to enable specific repositories, and another to assign a content view and lifecycle environment.
```

Since global registration is now the supported registration method and `subscription-manager register` is no longer recommended, I think we can leave off the command.